change/@ni-nimble-components-39e27099-91fe-44c4-8501-23b6e9f4c012.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Should we make similar changes for `tree-item` and `anchor-tree-item`?

Reply:
We should revisit the disabled styling of the tree item components because I don't think they are correct (simply applying 50% opacity to the whole item, icon and all, when disabled). However, I didn't see any visual designs that showed what the disabled state of a tree item should be. I'm guessing they should be styled the same as a disabled menu item, but I didn't want to clutter this PR with other changes that I wasn't sure about.

We probably need to reach out to designers and double check that the same pattern from the menu items applies to the tree items.